extensions/VisualEditor/wikia/modules/ve/init/ve.init.wikia.ViewPageTarget.js

```diff
@@ -73,16 +73,6 @@ ve.init.wikia.ViewPageTarget.static.toolbarGroups = [
 	}
 ];
 
-ve.init.wikia.activateToolbarABTest();
-
-if ( ve.init.wikia.getToolbarABTestVariantNumber() === 1 ) {
```

Comment:
Since we're removing the AB test, I think we should cleanup/remove all the references to the toolbar AB test (i.e. in the track and optimizely modules as well).